services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -428,6 +437,76 @@ async def pull_input_ports(
     return transferred_bytes
 
 
+@containers_router.post(
+    "/containers/directory-watcher:disable",
+    summary="Disable directory-watcher event propagation",
+    response_model=None,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def disable_directory_watcher(
+    app: FastAPI = Depends(get_application),
+) -> Response:
+    directory_watcher.disable_directory_watcher(app)
+    # SEE https://github.com/tiangolo/fastapi/issues/2253
+    return Response(status_code=status.HTTP_204_NO_CONTENT)
+
+
+@containers_router.post(
+    "/containers/directory-watcher:enable",
```

Comment:
https://google.aip.dev/156

Reply:
I guess this will be merged with the above api